components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useState } from 'react';
+import { ArrowLeft, ArrowRight } from 'react-feather';
+
+import { PaginationProps } from '@/components/Pagination/Pagination.types';
+
+const Pagination = ({
+  current,
```

Comment:
i think there should be an object for pagination params

`params: {
 current,
 limit, 
 total
} `

Reply:
I'm adding it to the to-do list.